### ISSUE TITLE
support for narval

### ISSUE THREAD

**Issue by kacperkan** [ubc-vision/compute-canada-goodies#12]

**Comment by kacperkan** [comment]
Add the following entry into `queue_cc.py`:

```python
    "narval":
        {
            "gpu_model": "a100",
            "gpus_per_node": 4,
            "cpu_cores_per_node": 48,
            "threads_per_node": 96,
            "cpu_cores_per_gpu": 12,
            "threads_per_gpu": 24,
            "ram_per_node": 510000,
            "ram_per_gpu": 127500,
            "job_system": "slurm",
            "default_account": "def-kyi-ab",
        },
```

**Comment by kmyi** [comment]
Can one of you create a PR and merge?

**Comment by kmyi** [comment]
closed by #13
